src/musl_missing.h

```diff
@@ -0,0 +1,65 @@
+/****************************************************************
```

Comment:
Instead of adding another file outside the tree, please consider adding the musl additions to src/basic/missing.h

However, musl_missing.h should go into src/basic and not src directly.